src/analysis/special_functions/trigonometric/bounds.lean

```diff
@@ -120,4 +123,44 @@ begin
   simpa only [tan_zero, sub_zero, sub_pos] using mono zero_in_U x_in_U h1
 end
 
+lemma le_tan {x : ℝ} (h1 : 0 ≤ x) (h2 : x < π / 2) : x ≤ tan x :=
+begin
+  rcases eq_or_lt_of_le h1 with rfl | h1',
+  { rw tan_zero },
+  { exact le_of_lt (lt_tan h1' h2) }
+end
+
+lemma cos_lt_one_div_sqrt_sq_add_one {x : ℝ}
+  (hx1 : -(3 * π / 2) ≤ x) (hx2: x ≤ 3 * π / 2) (hx3 : x ≠ 0) :
+  cos x < 1 / sqrt (x ^ 2 + 1) :=
+begin
+  suffices : ∀ {y : ℝ} (hy1 : 0 < y) (hy2 : y ≤ 3 * π / 2), cos y < 1 / sqrt (y ^ 2 + 1),
+  { rcases lt_or_lt_iff_ne.mpr hx3.symm,
```

Comment:
after this line you have two goals. This should be reflected in braces structure.